changes.md

```diff
@@ -1,3 +1,53 @@
+
+**Changelog location:** Starting next week we will post changelogs to the main [GitHub changelog](https://github.blog/changelog) rather than this repository. This will be the last changelog posted in this repository, so make sure to follow the [GitHub changelog](https://github.blog/changelog) to stay up to date with the latest and greatest.
+
+**How to provide feedback:** We are also transitioning from the current Discord to a [GitHub Discussion](https://gh.io/workspace-feedback) as the primary place for feedback and discussions around Copilot Workspace. We will still be available in Discord, but posting in the [discussion](https://gh.io/workspace-feedback) will ensure we see your feedback.
+
+Okay, no more logistics, now onto the product changes for this week! :tada:
+
+- [Image Preview Support](#Image-Preview-Support)
+- [Simplifying the Experience](#simplifying-the-experience)
+    - [Reducing Action Button Clicks](#reducing-action-button-clicks)
+    - [Consolidating the Plan Action Buttons](#consolidating-the-plan-action-buttons)
+- [VS Code Extension Improvements](#vs-code-extension-updates)
+
+### Image Preview Support
+
+Building on recent improvements to file and image support, you can now preview images directly in the Workspace editor. Selecting an image from the file tree will now display a full preview of the image, letting you open a preview tab directly within the editor.
+
+<img src="https://github.com/user-attachments/assets/939410dd-2b3a-488e-8d0d-558e9d33ae29" width="900px" >
+
+### Simplifying the Experience
+
+Since our last changes dropped we have invested time into streamlining the Workspace experience, saving you clicks, headaches, and frustration.
+
+#### Reducing Action Button Clicks
+
+We updated the primary action button such that secondary actions available in the dropdown no longer require a second click of the primary button - when you select an action it will immediately take effect.
+
+<img src="https://github.com/user-attachments/assets/95228cfd-3031-49b8-8e0f-d47b5c98d4ae" width="400" >
+
+#### Consolidating the Plan Action Buttons
+
+We have also consolidated plan action buttons like Regenerate and Add File to a kebab menu.
+
+**Before:**  
+<img src="https://github.com/user-attachments/assets/d049b613-b48a-4494-80eb-a4a92f1c00dd" width="400" >
+
+**After:**  
+<img src="https://github.com/user-attachments/assets/2f5a89f6-00c7-42cc-81aa-381f928cb34c" width="400" >
+
+### VS Code Extension Updates
+
+- **Stale View Fix:** Resolved an issue where stale view states were retained in certain views.
+- **Push to Branch / PR Creation Fix:** Fixed failures when merging into an existing branch with updates to the same files.
+- **Binary Detection Fix:** Addressed a false positive issue where folders were incorrectly flagged as binary after session syncing stopped.
+- **Enhanced Session List:** Sessions now appear earlier in their lifecycle in the session list, supporting the new brainstorming feature in VS Code.
```

Comment:
Should we pull this up above the bug fixes?

Reply:
Yeah that's a good call